docs/home.md

```diff
@@ -93,6 +93,14 @@ and working with the results.
 - **[What to do with DataFrame Results](df-results.md)**
 - **[What to do with RDD Results](rdd-results.md)**
 
+### Citing Archives Unleashed
+
+How to cite the Archives Unleashed Toolkit or Cloud in your research:
+
+> Nick Ruest, Jimmy Lin, Ian Milligan, and Samantha Fritz. 2020. The Archives Unleashed Project: Technology, Process, and Community to Improve Scholarly Access to Web Archives. In _Proceedings of the ACM/IEEE Joint Conference on Digital Libraries in 2020 (JCDL '20)_. Association for Computing Machinery, New York, NY, USA, 157–166. DOI: [https://doi.org/10.1145/3383583.3398513](https://doi.org/10.1145/3383583.3398513).
```

Comment:
Oh I approved this too early. This is throwing a line length error. You'll need to break into different lines.